USBPcapCMD/cmd.c

```diff
@@ -1064,102 +1106,73 @@ int print_extcap_options(const char *device)
            "{display=Capture from newly connected devices}"
            "{tooltip=Automatically start capture on all newly connected devices}"
            "{type=boolflag}{default=true}\n");
+    /*
+    TODO: Sometimes wireshark call --devices without argument. Investigate why.
```

Comment:
The multicheck doesn't work properly in Qt interface. I have checked this in Wireshark 2.0.16 - the GTK+ interface works as expected, however the Qt doesn't even allow selecting the devices. This is to be fixed in Wireshark Qt interface itself.